config/mocks/wallet-options-v4.json

```diff
@@ -47,12 +47,14 @@
         "mergeAndUpgrade": false,
         "nftExplorer": true,
         "recurringBuys": true,
+        "rewardsFlowUnderSwapEnabled": true,
```

Comment:
This is what I added, the rest of this file modifications is the alphabetical order.